_jobs/python-software-developer.md

```diff
@@ -0,0 +1,95 @@
+
+To communicate with Nuvla.io, edge devices run the NuvlaEdge software.  Packaged as micro-services (e.g. Docker and Kubernetes), NuvlaEdge provides the local environment that allows Nuvla.io to manage fleets of edge devices, as well as the life-cycle of business apps running at the edge.
+
+The core team is based in Geneva, in beautiful Switzerland. Remote working is possible, however you must be living in Europe or Switzerland, with the ability to come to Geneva from time to time (3-4 times a year). As the company operates using mature agile methodologies, experience or interest in agile development is required, since this is how we think and do things.
+
+Expect to be given time and space to learn before jumping right in developing production code for NuvlaEdge. As an agile team, we expect each developer to be able to contribute to feature definition, architecture, design, implementation, testing and operations support.  Also, expect to be presenting sharing your work to your pears, NuvleEdge Tech Lead, Scrum Master and Product Owner for feedback and guidance.
+
+Since we continue to operate as a scale-up, with the backing of a publicly listed company, you will have the opportunity to thrive in our fast moving, and exciting work environment.
+
+
+### Tasks and Activities
+
+The responsibilities will include:
+
+- production software development including designing solutions, writing code, testing code, automating test and deployment
+- delivery of software components working in collaboration with the rest of the team and integrated into our CI/CD pipeline
+- involvement in R&D projects, both in Switzerland and other European countries
+
+
+### Skills and Experience
+
+We are looking for candidates with the following skills and experience:  
+
+
+**Mandatory**
+
+- 3-5 years experience as software developer
+- Excellent Python skills
+- Experience in developing production code
+- Able to enunciate simple designs from high-level user story requirements, alone and in a small team
+- Demonstrated experience in working in a multi-cultural environment
+- Ability to communicate effectively with internal teams, partners and customers
+- Experience or interest in agile methodologies (e.g. Scrum, Kanban, DevOps)
+- Experience in Docker and/or Kubernetes
+- Experience of a shell scripting language (e.g. bash)
+- Good knowledge of Linux (NuvlaEdge micro-services are often based on Ubuntu)
+
+
+**Nice to have**
+
+- Experience with micro-service architecture (NuvlaEdge is a set of Docker images orchestrated using DockerSwarm or Kubernetes)
+- Experience with embedded system development
+- Knowledge of service operations in cloud environments - e.g. Exoscale, AWS, Azure, GCE
```

Comment:
Suggestion to change: "e.g. Exoscale, AWS, Azure, GCE" -> "e.g. AWS, Azure, GCE, CloudStack, OpenStack"